ui/app/controllers/optimize/summary.js

```diff
@@ -0,0 +1,12 @@
+import Controller from '@ember/controller';
+import { inject as controller } from '@ember/controller';
+
+export default class OptimizeController extends Controller {
```

Comment:
Oh, this makes the code above make more sense now. Can you override the getter here rather than using an if/else in the optimize controller?